apps/extension/src/features/accounts/account-card.tsx

```diff
@@ -0,0 +1,376 @@
+                            style={{ backgroundColor: darkColor }}
+                          ></div>
+                          <div
+                            className="checkmark_stem"
+                            style={{ backgroundColor: color }}
+                          ></div>
+                          <div
+                            className="checkmark_kick"
+                            style={{ backgroundColor: color }}
+                          ></div>
+                        </span>
+                      </div>
+                    )} */}
+                </div>
+              </RadioGroup.Root>
+
+              {/* {updating && <Spinner />} */}
+              {/* </div> */}
+            </div>
+          </div>
+          {/* Actions */}
+          <DropdownMenuDemo account={account} container={container!} />
+        </div>
+        {/* <div>
+          <div className="flex items-center space-x-5">
+            {account.isPrimary && (
+              <Tooltip.Root delayDuration={300}>
+                <Tooltip.Trigger>
+                  <div className="text-white bg-[#002F76] text-xs px-2 py-1 rounded cursor-help">
+                    Organizer
+                  </div>
+                </Tooltip.Trigger>
+                <Tooltip.Content align="end" side="bottom" sideOffset={10}>
+                  <div
+                    className="px-5 py-6 bg-white rounded-lg shadow-2xl"
+                    style={{ width: "350px" }}
+                  >
+                    <div className="font-semibold">Organizer Account</div>
+                    <div className="pt-2">
+                      The selected account will be the organizer for all events
+                      created.
+                    </div>
+                  </div>
+                </Tooltip.Content>
+              </Tooltip.Root>
+            )}
+            {optionsDropdown}
+          </div>
+        </div> */}
+      </div>
+      <div ref={setContainer} className="relative z-10"></div>
+      {/* <Modal
+        open={showDeleteModal}
+        title="Are you sure you want to remove this calendar from AgreeTo?"
+        primaryButton={{
+          text: "Remove",
+          onClick: handleDelete,
+          type: "danger",
+        }}
+        cancelButton={{
+          text: "Cancel",
+          onClick: () => setShowDeleteModal(false),
+        }}
+      /> */}
+    </Tooltip.Provider>
+  );
+};
+
+// why does this break the app?
+const DropdownMenuDemo = ({
+  account,
+  container,
+}: {
+  account: Account;
+  container: HTMLDivElement;
+}) => {
+  const utils = trpcApi.useContext();
+
+  const { mutate: changePrimary } = trpcApi.account.changePrimary.useMutation({
+    onSuccess() {
+      utils.user.myAccounts.invalidate();
+    },
+  });
+  const { mutate: removeAccount } = trpcApi.account.delete.useMutation({
+    onSuccess() {
+      utils.user.myAccounts.invalidate();
+    },
+  });
+  return (
+    <DropdownMenu.Root>
```

Comment:
maybe this should be a general purpose component in the ui package?

that would be usable for other components where we use the OutsideClickHandler + Headless Float combination?

<img width="426" alt="CleanShot 2022-11-23 at 00 30 46@2x" src="https://user-images.githubusercontent.com/51714798/203441779-c04d7f11-965f-40c6-b070-6dd348dd7f5b.png">

Reply:
Yes, it should. I agree. 🙌

Reply:
I've extracted the DropdownMenu into ui

It doesn't have a lot of variants yet though.